src/engraving/rw/write/twrite.cpp

```diff
@@ -2577,8 +2577,8 @@ void TWrite::write(const Staff* item, XmlWriter& xml, WriteContext& ctx)
     if (item->hideSystemBarLine()) {
         xml.tag("hideSystemBarLine", item->hideSystemBarLine());
     }
-    if (item->mergeMatchingRests()) {
-        xml.tag("mergeMatchingRests", item->mergeMatchingRests());
+    if (item->mergeMatchingRests() != AutoOnOff::AUTO) {
+        xml.tag("mergeMatchingRests", static_cast<int>(item->mergeMatchingRests()));
```

Comment:
Probably better to use `TConv::toXml(item->mergeMatchingRests())`. Writing enum values by casting to int and then writing that int is risky. Of course, the values of a simple generic enum like AutoOnOff are unlikely to change in practice, but still, better safe than sorry... 
(When writing other AutoOnOff properties, we also use `TConv::toXml`)